[DOCS-6621] Add link to parsing dates doc (#20792)
DataDog/documentation: content/en/logs/log_configuration/pipelines.md

### Processors

A processor executes within a pipeline to complete a data-structuring action. See the [Processors docs][3] to learn how to add and configure a processor by processor type, within the app or with the API. 

> See [Parsing dates][10] on how to parse a custom date and time format and for information on the `timezone` parameter, which is needed if your timestamps are not in UTC.

Reviewer: ```suggestion
See [Parsing dates][10] for more information about parsing a custom date and time format and for information on the `timezone` parameter, which is needed if your timestamps are not in UTC.
```
